torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -311,6 +311,34 @@ RegisterOperators reg(
            pack(stack, t.sizes().vec());
          },
          aliasAnalysisFromSchema()),
+     Operator(
+         "prim::EnumName(AnyEnumType enum) -> str",
+         [](Stack* stack) {
+           IValue e = pop(stack);
+           push(stack, e.toEnumHolder()->name());
+         },
+         aliasAnalysisFromSchema()),
+     Operator(
+         "prim::EnumValue.int(AnyEnumType enum) -> int",
```

Comment:
This is at least worth a discussion, because the duplication of the kernel has some binary implications. However, I think not-standard ops (unschematized ops) have some other complications around mobile and stuff so I think this is probably worth it.  @ljk53 what do you think ?
 
@SplitInfinity

Reply:
What is the "binary implication" you are referring to with dup kernels?

Reply:
Each operator increases the size of the mobile build (if they're included in a mobile model). In this case, we could unify all of the prim::EnumValue operators but at the cost of having to make it a non-standard op where the schema doesnt express its true types. I think our current decision is good.